src/node/element/path/parameters.rs

```diff
@@ -39,6 +39,13 @@ impl From<Parameters> for Vec<Number> {
     }
 }
 
+impl<'a> From<&'a mut Parameters> for &'a mut Vec<Number> {
```

Comment:
I think I use `'l` for lifetime everywhere in the package. Please let us this to that convention.

Reply:
OK, done.